### PR TITLE
docs: Encourage native OpenTelemetry intake use

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

* Remove experimental limitation from native OTLP
* Remove self-managed limitation from native OTLP (per https://github.com/elastic/apm/issues/212)
* Move native OTLP above elastic exporter to encourage use
* Move tab-widget code to the bottom of the page

### CODE REVIEW

File: docs/guide/opentelemetry-elastic.asciidoc [hunk 1]
```diff
@@ -164,76 +213,22 @@ java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
...
-[float]
 [[open-telemetry-elastic-traces-metrics]]
 ==== Collect traces and metrics
 
-To export traces and metrics, ensure that you have instrumented your services and applications
-with the OpenTelemetry API, SDK, or both.
-
-Here is an example of how to capture business metrics from a Java application.
+You're now ready to export traces and metrics from your services and applications.
+Here's an example of how to capture business metrics from a Java application.
```
Comment (bmorelli25) [inline docs/guide/opentelemetry-elastic.asciidoc:231]: I have a question about this section. You can see it live, [here](https://www.elastic.co/guide/en/apm/get-started/current/open-telemetry-elastic.html#open-telemetry-elastic-traces-metrics). What's the purpose of this section? Should we be linking to the OTEL Metrics API for more information?

The original section says:
> To export traces and metrics, ensure that you have instrumented your services and applications with the OpenTelemetry API, SDK, or both.

But that same line is also said in the two instrument sections:
* [instrument 1](http://localhost:8001/guide/open-telemetry-elastic.html#instrument-apps-collector)
  > To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to instrument your Java app using the OpenTelemetry agent for Java.
* [instrument 2](http://localhost:8001/guide/open-telemetry-elastic.html#instrument-apps-apm-server)
  > To export traces and metrics to APM Server, ensure that you have instrumented your services and applications with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to instrument your Java app using the OpenTelemetry agent for Java.

How we better clarify here?

**Comment by cyrille-leclerc** [comment]
Super thanks @bmorelli25 . We are likely to iterate on the phrasing regarding the OpenTelemetry Collector Exporter for Elastic to indicate that we "don't recommend it" as it' doesn't have the latest improvements that we have shipped in the native OTLP support (ie in APM Server).

File: docs/guide/opentelemetry-elastic.asciidoc [hunk 2]
```diff
@@ -25,20 +22,72 @@ business KPIs and technical components with the {stack}.
...
+     com.mycompany.checkout.CheckoutServiceServer
+----
+
+|===
+
+| `OTEL_RESOURCE_ATTRIBUTES` | The service name to identify your application.
+
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | APM Server URL. The host and port that APM Server listens for events on.
+
+| `OTEL_EXPORTER_OTLP_HEADERS` | Authorization header that includes the Elastic APM Secret token or API key: `"Authorization=ApiKey api_key"`.
```
Comment (EamonnTP) [inline docs/guide/opentelemetry-elastic.asciidoc:64]:
```suggestion
| `OTEL_EXPORTER_OTLP_HEADERS` | Authorization header that includes the Elastic APM Secret token or API key: `"authorization=ApiKey api_key"`.
```

File: docs/guide/opentelemetry-elastic.asciidoc (hunk 1 above)
Comment (EamonnTP) [reply docs/guide/opentelemetry-elastic.asciidoc:231]: I think the purpose was to show an example that included the usage of a custom metric: `order_value`. As you say, maybe we should link to the OTEL Metrics API (if we support all the metrics now).  Or you could include an example in each *instrument* section? @cyrille-leclerc What's your thoughts on this?

**Review by EamonnTP — APPROVED** [review]
Looks great! Just a couple of suggestions.

File: docs/guide/opentelemetry-elastic.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/guide/opentelemetry-elastic.asciidoc:231]: Ahh, ok, that makes sense. Added a link in 13f4961.

**Pull request merged**